### PR TITLE
KAA-439: Event FQNs are available in kaa_fqn_definitions.h.

### PR DESCRIPTION
**Pull request by ohamov**

### CODE REVIEW

File: client/client-multi/client-c/src/kaa/kaa_fqn_definitions.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2014-2015 CyberVision, Inc.
```
Comment (ghost) [inline client/client-multi/client-c/src/kaa/kaa_fqn_definitions.h]: Please, move this header to the gen directory and fix the path used by the SDK generator accordingly.

File: examples/c-sdk-event-demo/README [hunk 2]
```diff
@@ -31,9 +31,13 @@ To run the demo application, do the following:
...
 list of third-party dependencies for the Kaa C SDK.
 2. Download and install the Kaa sandbox (https://docs.kaaproject.org/display/KAA/sandbox).
-3. Generate the C SDK using the Kaa sandbox. The Avro log schema required for the demo application is located in 
-the 'avro' directory. Put the SDK archive into the 'libs/kaa' directory.
-4. Use the build.sh script to build and run the demo application (the 'deploy' option).
+3. Before generating C SDK, please make sure that you provided SDK with User verifier and following fields for Event class family were specified as follows:
+ Name: ThermostatEventClassFamily
+ Namespace: org.kaaproject.kaa.schema.sample.event.thermo
+ Class name: ThermostatEventClassFamily
+ (The Avro log schema required for the demo application is located in the 'avro' directory)
```
Comment (ghost) [inline examples/c-sdk-event-demo/README]: "The Avro event schema..." instead

File: client/client-multi/client-c/src/kaa/kaa_fqn_definitions.h [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+static const event_class_family_t SUPPORTED_EVENT_CLASS_FAMILIES[SUPPORTED_EVENT_CLASS_FAMILIES_SIZE] =
+{
+    {
+        /* .ecf_name = */                       "TestEventFamily",
+        /* .supported_incoming_fqns_count = */  8,
+        /* .supported_incoming_fqns = */        (char **)TestEventFamilyFQNS
+    }
+};
+# endif
```
Comment (izeigerman) [inline client/client-multi/client-c/src/kaa/kaa_fqn_definitions.h]: I guess this file should be renamed to "kaa_event_fqn_definitions.h"

**Comment by ghost** [comment]
LGTM

**Comment by izeigerman** [comment]
LGTM

**Pull request merged**